libs/components/forms/src/lib/modules/shared/form-field-label-text-required.directive.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  ChangeDetectorRef,
+  Directive,
+  HostBinding,
+  Input,
+  OnInit,
+  inject,
+} from '@angular/core';
+
+import { SkyFormFieldLabelTextRequiredService } from './form-field-label-text-required.service';
+
+/**
+ * Hide form field components of field groups that are missing label text.
+ * @internal
+ */
+@Directive({
+  selector: '[skyFormFieldLabelTextRequired]',
+  standalone: true,
+})
+export class SkyFormFieldLabelTextRequiredDirective implements OnInit {
+  @Input()
+  public labelText: string | null | undefined;
```

Comment:
The setter didn't work when the `labelText` attribute wasn't set at all. I moved its logic to the `ngOnInit` method.